x/ecocredit/server/helpers.go

```diff
@@ -37,7 +35,7 @@ func setDecimal(store sdk.KVStore, key []byte, value math.Dec) {
 	}
 }
 
-func getAddAndSetDecimal(store sdk.KVStore, key []byte, x math.Dec) error {
```

Comment:
i felt this name was kinda strange, as the `get` usually implies it would return the decimal